stdlib/src/builtin/string.mojo

```diff
@@ -51,6 +51,25 @@ fn _ctlz(val: SIMD) -> __type_of(val):
     return llvm_intrinsic["llvm.ctlz", __type_of(val)](val, False)
 
 
+@always_inline
+fn unicode_compare[T: Stringable](str1: T, str2: T) -> Int:
```

Comment:
The name of this function should not mention unicode as it is a pure utf8 byte comparison. Unicode defines a very complex set of rules for comparison, which is called collation and is described in following [document](http://www.unicode.org/reports/tr10/tr10-41.html).

Reply:
Thanks for your rigorous comments, I will modify this function name😊